packages/back-end/src/api/openapi/schemas/ExperimentResultsRow.yaml

```diff
@@ -0,0 +1,157 @@
+type: object
+required:
+  - experimentName
+  - experimentId
+  - snapshotId
+  - dimensionId
+  - dimensionName
+  - dimensionValue
+  - metricName
+  - metricId
+  - baselineVariationId
+  - variationName
+  - variationId
+  - variationUsers
+  - variationNumerator
+  - variationDenominator
+  - variationMean
+  - variationStdDev
+  - differenceType
+  - effect
+  - chanceToBeatControl
+  - relativeRisk
+  - pValue
+  - pValueAdjusted
+  - ciLower
+  - ciUpper
+  - ciLowerAdjusted
+  - ciUpperAdjusted
```

Comment:
Probably out of scope here, but how are you thinking about health tab data?  If I'm going to be doing a meta-analysis across a bunch of experiments, I'm definitely going to want to be able to filter on ones with/without an SRM or multiple exposures error.

Reply:
I wasn't considering it, but yeah we could also include the SRM p-value and SRM status as two columns, as well as number of multiple exposures.